prusti-common/src/vir/mod.rs

```diff
@@ -4,21 +4,15 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-pub use self::ast::*;
-pub use self::cfg::*;
-pub use self::conversions::*;
 pub use self::to_viper::*;
 pub use self::program::*;
+pub use self::to_graphviz::*;
```

Comment:
Is it necessary to import everything from this module?

Reply:
Thanks, updated.